data/fields/books.json

```diff
@@ -0,0 +1,5 @@
+{
+    "key": "books",
+    "type": "combo",
```

Comment:
this should probably a `semiCombo` field, for when different genres are offered

Reply:
Sure.